ivadomed/config/config.json

```diff
@@ -1,22 +1,22 @@
 {
     "command": "train",
-    "gpu": 7,
+    "gpu": 0,
     "log_directory": "spineGeneric",
     "model_name": "seg_sc_t1_t2_t2s_mt",
     "debugging": false,
     "object_detection_params": {
         "object_detection_path": null
     },
     "loader_parameters": {
-        "bids_path": "../duke/projects/ivadomed/spineGeneric_201907041011/result/",
+        "bids_path": "../dataset/data_spinegeneric_registered-master",
```

Comment:
What about creating a folder named 'tutorial_data' that would be downloaded for some user, and therefore use this path as an example?  This way the user doesn't have to change it ?

Reply:
Sorry I am not sure to understand.
Tutorial data: https://github.com/ivadomed/data_spinegeneric_registered
--> which is download by the user willing to do the tutorial by using the curl command indicated in this tutorial.

The config files in `config` are examples of configuration files for user-cases.
Currently we are also using some of them for our own experiments.. but we would like to change this: on your local machine you will have your own config file, while the ones on the repo' are more like "templates".

Reply:
Thanks to Andreanne-Translator I think I better understand what you meant, sorry about that!
I agree that we should make clear in the tutorial that this path is located where the user download the data! I am going to have a try... will submit you the related doc'! Excellent point, thanks!!

Reply:
Regarding the folder where they download the data: I feel they can make their own choice. I am not sure why we would want to enforce this.

Reply:
Done here [0a7034d](https://github.com/ivadomed/ivadomed/pull/309/commits/0a7034de026ad2d2b1c3634abb1599ca6976507a)